config/locales/en/claims/schools/users.yml

```diff
@@ -29,7 +29,7 @@ en:
           page_title: Are you sure you want to remove this user? - %{user_name}
           caption: "%{user_name}"
           heading: Are you sure you want to remove this user?
-          warning: The user will be sent an email to tell them you removed them from %{school_name}.
+          warning: "%{user} will be sent an email to tell them you removed them from %{school_name}."
```

Comment:
Can we use `%{user_name}` for the key here please? It's explicit and matches the other keys we have. :)